etc/ci/check_dynamic_symbols.py

```diff
@@ -0,0 +1,26 @@
+#/usr/bin/env python
+
+import sys, re, subprocess
+from sets import Set
```

Comment:
Also, I think a `frozenset` would be appropriate for the `allowed_symbols` variable.